src/platform/intel/ace/platform.c

```diff
@@ -80,7 +81,9 @@ int platform_init(struct sof *sof)
 
 	trace_point(TRACE_BOOT_PLATFORM_CLOCK);
 	platform_clock_init(sof);
-	clock_set_freq(CLK_CPU(cpu_get_id()), CLK_MAX_CPU_HZ);
+	ret = core_kcps_adjust(cpu_get_id(), CLK_MAX_CPU_HZ / 1000);
```

Comment:
so IIUC correctly, this sets max clock at init using kcps API now ? Maybe an inline comment above to state the reason and any expectation to have driver lower this when its ready.

Reply:
@lgirdwood yes, you're correct. Inline comment added.